src/data_distribution/global_ptr.hpp

```diff
@@ -101,19 +101,15 @@ namespace argo {
 				 * @brief return the home node of the value pointed to, or a
 				 * default value if the page has not yet been first-touched
 				 * under first-touch allocation.
-				 * @return home node id
+				 * @return home node id, or argo::data_distribution::invalid_node_id
+				 * if access_ptr has not been first-touched yet
```

Comment:
This comment is slightly incorrect on non-first-touch policies, because it will not return `invalid_node_id`

Reply:
This now mentions that the alternative return value only concerns first-touch allocation. Same concept applied below.